### PR TITLE
bump version: Netlify functions, Svelte kit & adapter-netlify

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
Bumped Versions for the following:

- netlify functions -> 1.2.0
- adapter-netlify -> 1.0.0-next.55
- sveltejs/kit -> 1.0.0-next.292

## Related Issue(s)

From Recent days (after https://github.com/gitpod-io/website/pull/2471 ) we have been facing many of build failures due to `sveltejs/adapter-netlify`.
This patch will fix it.

## How to test
- Open Netlify Preview URl

## Release Notes
```release-note
NONE
```

## Documentation
No

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2681"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2681"></a>

### CODE REVIEW

**Review by Cahllagerfeld — COMMENTED** [review]
Its hard to approve it from the distance. As its a breaking change, please test all features, especially the Forms very carefully beofre merging

**Comment by Siddhant-K-code** [comment]
> Its hard to approve it from the distance. As its a breaking change, please test all features, especially the Forms very carefully beofre merging

Feedback forms are working perfectly 💯 
I do not know, how to test sales/ support forms
Can you please tell me about it?

**Comment by Cahllagerfeld** [comment]
Send an email with the forms, and then you need to check if the email reaches the google group for sales and contact

**Comment by Siddhant-K-code** [comment]
> google group for sales and contact

Can you please do it for me, I do not have access to that.

File: tsconfig.json [hunk 1]
```diff
@@ -1,31 +1,17 @@
...
-			"$lib/*": ["src/lib/*"]
-		}
-	},
-	"include": ["src/**/*.d.ts", "src/**/*.js", "src/**/*.ts", "src/**/*.svelte"]
+		"esModuleInterop": true,
+		"forceConsistentCasingInFileNames": true,
+		"resolveJsonModule": true,
+		"skipLibCheck": true,
+		"sourceMap": true,
+		//strict: true,
```
Comment (Cahllagerfeld) [inline tsconfig.json:11]: had to disable that, otherwise there would be around 300 type errors. That's better done with a followup PR

Comment (Siddhant-K-code) [reply tsconfig.json:11]: Rest all things are working fine, is it good to go?

**Review by Cahllagerfeld — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (Siddhant-K-code) [reply tsconfig.json:11]: so, in follow-up PR, you want to make it to `strict: true` (_basically uncomment Line 11_)?

Comment (Cahllagerfeld) [reply tsconfig.json:11]: Yep, it's going to be a bigger effort, I guess as there are quite a few typescript-issues (they aren't breaking the build though)

Comment (Siddhant-K-code) [reply tsconfig.json:11]: I am merging it 🙏🏼

**Pull request merged**